src/status_im2/contexts/onboarding/identifiers/profile_card/view.cljs

```diff
@@ -0,0 +1,73 @@
+(ns status-im2.contexts.onboarding.identifiers.profile-card.view
+  (:require [react-native.core :as rn]
+            [react-native.hole-view :as hole-view]
+            [react-native.reanimated :as reanimated]
+            [react-native.fast-image :as fast-image]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.markdown.text :as text]
+            [quo2.components.profile.profile-card.style :as profile-card-style]
```

Comment:
imo I don't really like to use styles across multiple views, especially since it is in quo2. Given this use case I would think it's better to just copy these styles into `status-im2.contexts.onboarding.identifiers.profile-card.style` and use from there.

Reply:
Copied styles 👍